<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compound Pi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListView" name="pi_list"/>
    </item>
    <item>
     <widget class="QListView" name="image_list">
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="servers_menu">
    <property name="title">
     <string>&amp;Servers</string>
    </property>
    <addaction name="find_action"/>
    <addaction name="add_action"/>
    <addaction name="remove_action"/>
    <addaction name="separator"/>
    <addaction name="configure_action"/>
   </widget>
   <addaction name="servers_menu"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="find_action"/>
   <addaction name="add_action"/>
   <addaction name="remove_action"/>
   <addaction name="separator"/>
   <addaction name="configure_action"/>
  </widget>
  <action name="find_action">
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="add_action">
   <property name="text">
    <string>&amp;Add...</string>
   </property>
  </action>
  <action name="remove_action">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="configure_action">
   <property name="text">
    <string>&amp;Configure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
